public/scripts/theme.js

```diff
@@ -20,7 +20,8 @@
     }
 })();
 
-function takePill(color) { // eslint-disable-line
+function takePill(color) {
+    // eslint-disable-line
```

Comment:
these lints may cause some errors

Reply:
> Am looking through it to try find root cause but as it stands it shows both the mobile nav and regular nav on desktops

I made the breakpoint higher for the mobile nav to be shown to accommodate the new page link. Right now on the live Pools site, the mobile nav icon still shows on desktop (under 1280px vs under 1536px), just at a lower breakpoint:
![image](https://user-images.githubusercontent.com/19278287/161647058-8024481c-2179-4809-8fd4-bcf05724f1db.png)

Reply:
^ in reply to the first comment "Am looking through it to try find root cause but as it stands it shows both the mobile nav and regular nav on desktops"

Reply:
true, this is bad. Am pretty busy right now but should definitely be fixed up